ci-operator/step-registry/ipi/conf/aws/oidc-creds-provision/ipi-conf-aws-oidc-creds-provision-commands.sh

```diff
@@ -120,7 +120,16 @@ if [[ "${FEATURE_SET}" == "TechPreviewNoUpgrade" ]]; then
 fi
 
 # create required credentials infrastructure and installer manifests
-ccoctl aws create-all ${CCOCTL_OPTIONS} --name="${infra_name}" --region="${REGION}" --credentials-requests-dir="/tmp/credrequests" --output-dir="/tmp"
+ccoctl_ouptut="/tmp/ccoctl_output"
+ccoctl aws create-all ${CCOCTL_OPTIONS} --name="${infra_name}" --region="${REGION}" --credentials-requests-dir="/tmp/credrequests" --output-dir="/tmp" &> "${ccoctl_ouptut}"
+cat "${ccoctl_ouptut}"
+
+# save oidc_provider info for upgrade
+oidc_provider_arn=$(grep "Identity Provider created with ARN:" "${ccoctl_ouptut}" | awk -F"ARN: " '{print $NF}')
+if [[ -n "${oidc_provider_arn}" ]]; then
+  echo "Saving oidc_provider_arn: ${oidc_provider_arn}"
+  echo "${oidc_provider_arn}" > "${SHARED_DIR}/aws_oidc_provider_arn"
```

Comment:
@jianlinliu Could we unify the file name to be `oidc_provider_arn` without platform info? Then upgrade step can read the file correctly.

Reply:
That was my original idea, but because when running ccoctl command in pre-upgrade steps, still need to run different command per different platforms, so I added platform info.

Reply:
make sense